packages/components/src/components/input/input.scss

```diff
@@ -58,7 +58,7 @@
             color: var(--text-disabled);
 
             & ~ label {
-                color: var(--text-disabled);
+                color: var(--text-disabled) !important;
```

Comment:
finalizing a style at the library level reduced the chance of customization, is it possible to handle this on the consumer side?

Reply:
I'm checking

Reply:
We must have this change for all input components, so it’s meaningful to have it in the library. 
Here there is no other way for changing the color of label but using `important` flag. If we want to change the color without `important` we need to have a lot of changes.

Reply:
So what happens now? if a consumer component wants to change the label color, what is the proper behavior? :thinking:

Reply:
We can add a class to the component and change the styles like this:
```.my-input-class {
    .dc-input__field:disabled ~ label {
        color: var(--brand-red-coral) !important;
    }
}```